drivers/NFSSR.py

```diff
@@ -137,19 +137,24 @@ def attach(self, sr_uuid):
         if not self._checkmount():
             self.validate_remotepath(False)
             util._testHost(self.dconf['server'], NFSPORT, 'NFSTarget')
-            io_timeout = util.get_nfs_timeout(self.session, sr_uuid)
-            self.mount_remotepath(sr_uuid, io_timeout)
+
+            #Extract timeout and retrans values, if any
+            sr_other_config = util.get_other_config(self.session, sr_uuid)
+            io_timeout = util.get_nfs_timeout(self.session, sr_uuid, sr_other_config)
+            io_retrans = util.get_nfs_retrans(self.session, sr_uuid, sr_other_config)
+            self.mount_remotepath(sr_uuid, io_timeout, io_retrans)
+            
         self.attached = True
 
 
-    def mount_remotepath(self, sr_uuid, timeout = 0):
+    def mount_remotepath(self, sr_uuid, timeout = 0, retrans = nfs.SOFTMOUNT_RETRANS):
         if not self._checkmount():
             # FIXME: What is the purpose of this check_server?
             # It doesn't stop us from continuing if the server
             # doesn't support the requested version. We fail 
             # in mount instead
             self.check_server()
-            self.mount(self.path, self.remotepath, timeout)
+            self.mount(self.path, self.remotepath, timeout, retrans)
```

Comment:
if we do not decide anything about timeout in the calling functions we need default values here as well..
Should we stick with our own default?